xarray/core/dataarray.py

```diff
@@ -1768,22 +1768,20 @@ def to_dict(self):
         Useful for coverting to json. To avoid datetime incompatibility
         use decode_times=False kwarg in xarrray.open_dataset.
 
+        Parameters
+        ----------
+        data : bool, optional
+            Whether to include the actual data in the dictionary. When set to
+            False, returns just the schema.
```

Comment:
It could be useful to allow including the data for the coordinates but not the data variables? 

I'm thinking something like 
```python
ds.to_dict(data='coords')
```